scripts/two_line_bash.sh

```diff
@@ -21,17 +29,25 @@ function parse_git_branch {
 	PS_BRANCH="(git ${ref#refs/heads/}) "
 }
 
+function parse_tlb_info {
+	parse_catkin_workspace
+	parse_git_branch
+}
+
+
 if [ ! -f ~/.disable_tlb ]; then
 	RESET="\[\033[0m\]"
 	RED="\[\033[0;31m\]"
 	GREEN="\[\033[01;32m\]"
 	BLUE="\[\033[01;34m\]"
 	YELLOW="\[\033[0;33m\]"
 
-	PS_LINE=`printf -- '- %.0s' {1..200}`
-	PROMPT_COMMAND=parse_git_branch
-	PS_INFO="$GREEN\u@\h$RESET:$BLUE\w"
+	PROMPT_COMMAND=parse_tlb_info
+	PS_LINE=`printf -- "- %.0s" {1..200}`
+	PS_FILL=${PS_LINE:0:$COLUMNS}
+	PS_INFO="$GREEN\u@\h$RESET:$BLUE\w "
+	PS_CATKIN="$YELLOW\$PS_WORKSPACE"
 	PS_GIT="$YELLOW\$PS_BRANCH"
 	PS_TIME="\[\033[\$((COLUMNS-10))G\] $RED[\t]"
-	export PS1="\${PS_FILL}\[\033[0G\]${PS_INFO} ${PS_GIT}${PS_TIME}\n${RESET}\$ "
+	export PS1="\${PS_FILL}\[\033[0G\]${PS_INFO}${PS_CATKIN}${PS_GIT}${PS_TIME}\n${RESET}\$ "
```

Comment:
`${PS_CATKIN}` should not be part of the prompt by default. There should be a way to include it on machines that actually have multiple workspaces.